### ISSUE TITLE
HIGH quality VRAM Compressed textures breaks Opengl 3 with ANGLE on Windows

### ISSUE THREAD

**Issue by IsaqueS** [godotengine/godot#95561]
### Tested versions

Tested on: Godot **4.3** _Release_ and Godot **4.2.2** _Release_

The project was not teste on godot 4.1 because the angle compatibility on windows was introduced on version 4.2

### System information

Godot v4.3.stable - Windows 11 - GLES3 (Compatibility) - NVIDIA GeForce RTX 2050 (NVIDIA; 32.0.15.6070) - 11th Gen Intel(R) Core(TM) i5-11400H @ 2.70GHz (12 Threads)

### Issue description

If your project uses any compressed texture where the high quality setting is enabled, if you use **OpenGl** with **ANGLE** on **Windows**, the texture will not load and the game will crash.

This issue seems to persist with both textures formats (S3TC BPTC and ETC2 ASTC)

This was tested on both recent computers (with rtx 2050), and also second generation intel cpus (2010-2011)

### Steps to reproduce

Create a new project, use the compatibility option, then import an image and select the "compressed texture" option and select the high quality option.

After that, run the project using Angle on Windows.

(I recommend exporting the project first so the editor doesn't crash)

### Minimal reproduction project (MRP)

**THE PROJECTS ARE SET TO USE THE NORMAL OPENGL MODE**

To see the bug on action, set to angle mode manually _(Reminder that this bug was tested on windows)_

[Godot 4.2.2 Opengl3 ANGLE PROJECT.zip](https://github.com/user-attachments/files/16626069/Godot.4.2.2.Opengl3.ANGLE.PROJECT.zip)
[Godot 4.3 Opengl Angle - Project.zip](https://github.com/user-attachments/files/16626070/Godot.4.3.Opengl.Angle.-.Project.zip)

**Comment by Calinou** [comment]
BPTC support should work in theory, as it's pretty much the *de facto* standard VRAM compression format in AAA games that use Direct3D 11 or 12.

I don't know if ASTC support is exposed in Direct3D 11, but note that it's not supported on desktop platforms in general, with the exception of Intel integrated graphics (**not** dedicated Arc GPUs).

**Comment by IsaqueS** [comment]
Yes, the game runs using Direct3D 11, thats what godot prints on the start of execution.

and I can confirm that it opened on a GPU without opengl 3 support, that is, only having the option of using ANGLE ( I5 from second Gen)

about the textures formats, to tell you the truth I don't know much about the differences between the two types, I just made sure to test both types using the project's export settings to make sure it wasn't that.

anyway, if it were possible to fix this it would be really important to me, because my dad's computer needs this translation layer to run the game, and it would be really cool if he could play my game too :)

**Comment by Calinou** [comment]
> anyway, if it were possible to fix this it would be really important to me, because my dad's computer needs this translation layer to run the game, and it would be really cool if he could play my game too :)

You could disable High Quality VRAM compression in the meantime. This means the texture will use S3TC compression on desktop platforms (ETC2 on mobile), which doesn't look as good, but provided the texture resolution is high enough, it'll probably be unnoticeable on most displays.

**Comment by BlueCube3310** [comment]
It looks like when using ANGLE, the `bptc` extension doesn't get registered for some reason. It seems to be officially supported, so it may be an issue on our end. A similar issue happens when running Linux with OpenGL-ES instead of OpenGL

**Comment by BlueCube3310** [comment]
#102782 fixes this for me, though it's intended to be more of a general solution
